channeldb/graph.go

```diff
@@ -1459,6 +1486,31 @@ func putLightningNode(nodeBucket *bolt.Bucket, aliasBucket *bolt.Bucket, node *L
 	return nodeBucket.Put(nodePub, b.Bytes())
 }
 
+func putPartialLightningNode(nodeBucket *bolt.Bucket, aliasBucket *bolt.Bucket, node *LightningNode) error {
+	var (
+		scratch [8]byte
+		b       bytes.Buffer
+	)
+
+	nodePub := node.PubKey.SerializeCompressed()
+
+	if err := aliasBucket.Put(nodePub, []byte(node.Alias)); err != nil {
```

Comment:
In the scenario that we're adding a partial node, then we don't yet know it's alias.